modules/@apostrophecms/translation/index.js

```diff
@@ -54,7 +58,15 @@ module.exports = {
             );
           }
 
-          return module.translate(req, providerName, doc, source, target, options);
+          return self.apos.modules['@apostrophecms-pro/automatic-translation']
+            .translate(req, {
+              providerName,
+              doc,
+              docType,
+              source,
+              target,
+              existing
+            });
```

Comment:
Just using the right module. Or rework the way we register the proxy and get it.